reana_workflow_controller/rest/utils.py

```diff
@@ -434,8 +434,16 @@ def list_directory_files(
     workspace_path: str, search: Dict[str, List[str]] = None
 ) -> List[dict]:
     """Return a list of files inside a given workspace."""
+    if int(os.getenv("REANA_GC_ENABLED", "0")) > 0:
```

Comment:
This does not have to do `os.getenv()` again; please import the value from the config file. That will be much faster.